src/main/java/stream/flarebot/flarebot/util/LocalConfig.java

```diff
@@ -1,131 +1,66 @@
 package stream.flarebot.flarebot.util;
 
-import com.google.gson.*;
+import com.google.gson.Gson;
+import com.google.gson.JsonParser;
+import org.json.JSONObject;
+import org.json.JSONTokener;
 import stream.flarebot.flarebot.FlareBot;
 
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.Reader;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Arrays;
+import java.util.stream.Collectors;
 
 public class LocalConfig {
 
     private final Gson gson = FlareBot.GSON;
     private final JsonParser parser = new JsonParser();
 
-    private JsonObject object;
+    private JSONObject object;
 
     public LocalConfig(URL url) {
 
         if (url == null)
             throw new IllegalArgumentException("URL cannot be null!");
 
         try (Reader r = new InputStreamReader(url.openStream())) {
-            JsonElement element = new JsonParser().parse(r);
-            if (element != null && !element.isJsonNull() && element.getAsJsonObject() != null)
-                this.object = element.getAsJsonObject();
-            else
-                this.object = new JsonObject();
+            JSONTokener tokener = new JSONTokener(r);
+            this.object = new JSONObject(tokener);
         } catch (IOException e) {
             FlareBot.LOGGER.error("There was an error reading the config file!", e);
         }
     }
 
-    public JsonElement getElement(String s) {
```

Comment:
👀 the modification should be minimal enough

Just refactor to be `getObject` or `getElement` and remove the cast at the end.